### DISCUSSION TITLE
Blocked kfx, can import .azw but not convert

### DISCUSSION THREAD

**Discussion by bookkat** [apprenticeharper/DeDRM_tools#1702] [Q&A]
I used sudo chmod -x (Method 2) with K4Mac 1.30.0 and I can now download books in azw format, import them into Calibre, but not convert them/remove DRM.

I am using tools v6.8 and Calibre 4.23 on Big Sur.

I followed the instructions here.
https://www.mobileread.com/forums/showpost.php?p=3819708&postcount=508

What am I missing?
I really appreciate your help. In exchange, I can offer translation, research, or writing/editing services.
I have been using Calibre and DeDRM for years and years and never had a problem until now. Argh.

**Comment by ElleKayEm** [comment]
Use DeDRM 6.8.1.

**Reply by bookkat** [reply]
IT WORKED. Ugh, can't believe that's what slipped by me: 6.8.0 vs 6.8.1. I had been turning around in circles for days.
Thank you so very much. <3
